src/assets/css/librarybrowser.scss

```diff
@@ -759,20 +763,16 @@
         margin-bottom: 0;
         padding-left: 0;
     }
+}
 
-    [dir="ltr"] & {
-        margin-left: 0;
-    }
-
-    [dir="rtl"] & {
-        margin-right: 0;
-        padding-left: unset !important;
+.layout-mobile [dir="rtl"] .mainDetailButtons {
+    margin-right: 0;
 
-        @include header-poster-padding-rtl;
+    @include header-poster-padding-rtl;
 
-        @media all and (max-width: 32em) {
-            padding-right: 0 !important;
-        }
+    @media all and (max-width: 32em) {
+        margin-bottom: 0;
+        padding-right: 0;
     }
 }
```

Comment:
Try the technique I suggested for `.infoWrapper'.

Reply:
Can't do it for this one because the linter complains. There is already another `.mainDetailButtons` up in the file.